hermes-common/src/main/java/pl/allegro/tech/hermes/domain/topic/schema/ZookeeperSchemaSourceProvider.java

```diff
@@ -0,0 +1,37 @@
+package pl.allegro.tech.hermes.domain.topic.schema;
+
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.zookeeper.KeeperException;
+import pl.allegro.tech.hermes.api.SchemaSource;
+import pl.allegro.tech.hermes.api.Topic;
+import pl.allegro.tech.hermes.common.exception.InternalProcessingException;
+import pl.allegro.tech.hermes.infrastructure.zookeeper.ZookeeperPaths;
+
+import java.nio.charset.Charset;
+import java.util.Optional;
+
+public class ZookeeperSchemaSourceProvider implements SchemaSourceProvider {
+
+    protected static final String SCHEMA_SUFFIX = "schema";
```

Comment:
I like when all zk paths are in one place - ZookeeperPaths class. Maybe path to schema also should be kept in this class and we could get it through getSchemaPath(). Thanks that you will not need to declare directly paths in tests. What do you think about it?